.github/workflows/test_adaptersv2.yaml

```diff
@@ -66,9 +66,9 @@ jobs:
           echo "value=${USER_INPUT:-"meshery"}" >> $GITHUB_OUTPUT
       - name: Setting up minikube
         id: setk8s
-        uses: manusa/actions-setup-minikube@v2.10.0
+        uses: manusa/actions-setup-minikube@v2.11.0
```

Comment:
Just a suggestion, the latest version `manusa/actions-setup-minikube` is `v0.2.13`. So should we consider using it?

Reference:  https://github.com/manusa/actions-setup-minikube/releases